docs/components/background.md

```diff
@@ -13,22 +13,35 @@ frustum culling issues when `a-sky` is further than the far plane of the
 camera. There are no unexpected occlusions either with far objects that might
 be behind of the sphere geometry of `a-sky`.
 
+The background component will ensure that the environment is generate using WebXR
+Lighting estimation in Augmented Reality. It will turn off the scene lights and provide
+it's own lights when AR is entered and will restore them when AR is exited.
+
+These lights are a `directional` and a `probe` light. If your scene makes use of shadows
+from a directional light you can provide this as the `directionalLight` property and it
+will control that light instead of making it's own one. Once the user leaves AR this
+light may have a different color and position than when they entered AR as it has been
+altered by the lighting estimation.
+
 The background component can also generate a default environment cube map for all
 materials, if you find GLB models end up too dark or reflective materials don't
 reflect the environment this will provide a default reflective environment.
 
 ## Example
 
-The example below sets the background color to red.
+The example below sets the background color to red and use lighting estimation for AR.
 
 ```html
-<a-scene background="color: red"></a-scene>
+<a-scene webxr="optionalFeatures: light-estimation;" background="color: red"></a-scene>
 ```
 
 ## Properties
 
-| Property            | Description                                               | Default Value |
-|---------------------|-----------------------------------------------------------|---------------|
-| color               | Color of the scene background.                            | black         |
-| transparent         | Background is transparent. The color property is ignored. | false         |
-| generateEnvironment | Whether to generate a default environment                 | true          |
+| Property                   | Description                                               | Default Value   |
+|----------------------------|-----------------------------------------------------------|-----------------|
+| color                      | Color of the scene background.                            | black           |
+| transparent                | Background is transparent. The color property is ignored. | false           |
+| generateEnvironment        | Whether to generate a default environment.                | true            |
+| environmentUpdateFrequency | How often to update the environment in seconds, 0 is off. | 0               |
+| sceneLights                | Query Selector for the lights to turn off when AR starts. | a-light,[light] |
```

Comment:
Does this need to be exposed on the component? It looks like it should stay internal?

Reply:
The only lights that you want to be turned off are the environment lights. Lights that are part of the scene such as a car's headights, a building's internal lights or some glowing features on a model should probably stay lit.

Reply:
I see. In this case all the lights that use `a-light` and `light` component will be disabled. Some of them could belong to the scene. Maybe we just want to disabled the env lights that A-Frame adds by default?

Reply:
I think lots of people use their own lights rather than relying on the default ones and this even works with the environment component. That someone will have models which include lights which need to be kept on is an edge case. 

Although I guess if a gltf model included lights they would be unaffected by this anyway since they won't get their own html element.

Reply:
Oh!  Maybe it should only disable lights which effect infinite areas such as ambient, directional, probe and hemisphere and leave all the others? 

That should leave all Point and Spot lights which only do small areas and are most likely to be the lights you want to leave on?